lib/react.dart

```diff
@@ -453,7 +464,7 @@ abstract class Component {
   /// > This will be completely removed alongside the Component class.
   @Deprecated('This legacy, unstable context API is only supported in the deprecated Component, and not Component2.'
       ' Instead, use Component2.context, Context.Consumer, or useContext.')
-  void componentWillUpdateWithContext(Map nextProps, Map nextState, Map nextContext) {}
+  void componentWillUpdateWithContext(Map nextProps, Map nextState, Map? nextContext) {}
```

Comment:
Is it expected that `nextContext` is nullable here, but not in `shouldComponentUpdateWithContext`?

Reply:
Good catch. It seems like they'll always be non-null, but for this deprecated API frankly I don't want to go through the trouble of making these non-nullable in a safe way, so I'm just gonna make `shouldComponentUpdateWithContext` nullable to match.

Reply:
Addressed in https://github.com/Workiva/react-dart/pull/373/commits/0616cd087f5433c117f09ceb83fcecf19e6665a1